src/otx/engine/engine.py

```diff
@@ -311,6 +311,8 @@ def test(
         if is_ir_ckpt and not isinstance(model, OVModel):
             datamodule = self._auto_configurator.update_ov_subset_pipeline(datamodule=datamodule, subset="test")
             model = self._auto_configurator.get_ov_model(model_name=str(checkpoint), label_info=datamodule.label_info)
+            if self.device.accelerator == "auto":
+                self.device = DeviceType.cpu  # type: ignore[assignment]
```

Comment:
If we always need to use the CPU, do we need an if statement?

Reply:
I'm also curious, when user give device gpu by mistake